rustfmt.toml

```diff
@@ -1,4 +1,5 @@
 imports_granularity = "Module"
 group_imports = "StdExternalCrate"
+edition = "2021"
```

Comment:
I believe you dont need to define this here. `rustfmt` automatically picks up the version in `Cargo.toml`.

Reply:
Oops, was going to ask something about that because my setup didn't like it being missing but I forgot